pyworkflow/plugin.py

```diff
@@ -52,6 +49,12 @@
 from pyworkflow.template import Template
 from pyworkflow.utils import sortListByList
 
+try:
+    import importlib_metadata
+except ModuleNotFoundError:
```

Comment:
Should we address this with conditional requirements?

I think import lib is backported. Not sure if we have installations in 3.6 or 3.7 though.

Reply:
https://docs.python.org/3/library/importlib.metadata.html
importlib_metadata is the backport of importlib.metadata. It does require python 3.8+, yes.